_data/guides.yaml

```diff
@@ -7,9 +7,41 @@ categories:
       - title: Getting Started with WildFly
         url: /get-started
         description: Build and run a Jakarta EE application with WildFly in a few minutes
-      - title: Java Microservices on Kubernetes with WildFly
-        url: /guides/get-started-microservices-on-kubernetes
-        description: Build and run Jakarta EE applications with WildFly on Kubernetes in a few minutes
+  - category: Cloud / Containerization
```

Comment:
Please put this block after Automation.

It's moved beyond any kind of 'Getting Started', so it should be alphabetical with the other categories.

Other than that, LGTM.

Reply:
@bstansberry done!